### ISSUE TITLE
[Feature Request] Show logo of core next to playlist entry

### ISSUE THREAD

**Issue by hyarsan** [libretro/RetroArch#5858]
Here is a thing I have been thinking about lately: Wouldn't it be cool if the logo of the Libretro core shows next to the names in the playlists? It would liven up some things don't you think? The logos would be stored in retroarch-assets. Maybe in a folder named "logos"? And it would probably be smart if the filenames of the logos are the same as the actual cores. ( For example, " ) Or maybe the logos could also be embedded into the core just in case it doesn't exist? Maybe the Libretro core could tell the frontend the location of the logo / icon? I also feel it should be in the place of the game cover art if it doesn't exist. It would also make it feel more like programs, because that's what Libretro cores really are :)

Badly made mockups:
![genius](https://user-images.githubusercontent.com/32988993/33580136-50cfdcf8-d94c-11e7-9198-3787393b1147.png)
![plzaddthx](https://user-images.githubusercontent.com/32988993/33580138-517309f0-d94c-11e7-9ffc-4d5ab23cb7c4.png)

**Comment by RobLoach** [comment]
The right-hand side of the options are usually used for the Thumbnail of the core:

![thumbs](https://user-images.githubusercontent.com/25086/33582134-c0abdf7a-d921-11e7-9fe4-8e438899e4b8.png)

I like the idea of replacing the Load Core icons with the XMB core playlist icons, however.

**Comment by hyarsan** [comment]
I have never seen such thing as a core thumbnail. But that's okay :)

**Comment by RobLoach** [comment]
Would be really cool to have the Load Core and Download Core icons show the Core icons :+1:

**Comment by hyarsan** [comment]
Oh that's true, it would make it easier to find the cores! I thought thumbnails were for box art and screenshot and title screen only? cool

**Comment by hyarsan** [comment]
And it should also have the logo when it asks for a core to open something after choosing content.

**Comment by fr500** [comment]
I think it would look very very very tacky

**Comment by bparker06** [comment]
Not a huge fan of core-specific icons myself, but what about generic system icons (like the playlists use)? Otherwise I think it's fine the way it is.

**Comment by hyarsan** [comment]
Whats wrong with a non-default toggle for it? I like it, RobLoach likes it so I bet others might like it too? Like RobLoach said: 

> I like the idea of **replacing** the Load Core and Download Core icons with the Core logos

Replacing the icons instead of putting it next to it looks a lot better.

**Comment by RobLoach** [comment]
The generic system ones from the XMB playlists would be really nice.

**Comment by hyarsan** [comment]
Welp, there goes any chance of this being implemented.

**Comment by twinaphex** [comment]
Well, one thing that I want to do at first, is replace the 'default' icon of the cores inside the Core list instead of the actual thumbnail.

**Comment by hyarsan** [comment]
Yeah, I agree with that it is not a good idea to to have it take up the space of the thumbnail.

**Comment by hyarsan** [comment]
Well, I am expanding on this idea... what if we have a different logo based on the menu icon theme? Like for example Citra is going to be monochrome when Monochrome is enabled. And I will make it for you lol

**Comment by thatman84** [comment]
@hyarsan did you ever get started making any of these?
